src/resource/search/dto/index.ts

```diff
@@ -0,0 +1 @@
+export * from './search.result.dto';
```

Comment:
I know it can be confusing. But try to keep the same 'name system' like in other dtos in other resouces.